README.md

```diff
@@ -50,7 +50,17 @@ Example for local development on the host **http://localhost:3000** and staging
 
 1.  Get the auth0 application values: **DOMAIN, CLIENT_ID, CLIENT_SECRET**
 
-1.  Create file **.env.local** and copy commented-out `AUTH0_*` variables from **.env**.
+1.  Create file **.env.local** with the following contents:
```

Comment:
`.env` has some uncommented vars now, so it might be worth removing a comment from there. It adds noise.